packages/sanity/src/core/preview/components/PreviewLoader.tsx

```diff
@@ -57,16 +57,25 @@ export function PreviewLoader(
 
   const uploadState = useMemo(() => _extractUploadState(value), [value])
 
+  const media: PreviewProps['media'] = useMemo(() => {
+    if (uploadState?.previewImage) {
+      return <img alt="The image currently being uploaded" src={uploadState.previewImage} />
+    }
+
+    if (!preview?.value?.media) {
+      return schemaType.icon
+    }
+
+    // @todo: fix `TS2769: No overload matches this call.`
+    return preview?.value?.media as any
```

Comment:
Not sure how to fix this type issue, weirdly it works when passing it to the component but when assigned to a variable it does not work. My guess is because `ReactNode` is a `type` composed of other types it has an issue when matching a type with the expanded out type

Reply:
yeah, something weird going on with the typings here, no worries.